README.md

```diff
@@ -14,6 +14,25 @@
 
 <br>
 
+- [CSCI 350 Operating Systems Image](#csci-350-operating-systems-image)
```

Comment:
Do you think this toc is more useful/explicit than the one github includes? People might not know about it but gh recently did add a toc menu bar to md files: 
![image](https://user-images.githubusercontent.com/17013462/117861395-3dd8b800-b246-11eb-9847-568ac62e5d58.png)

Reply:
Well I didn't know gh has toc baked in either LOL so maybe it's fine until more people start to know it exists